packages/ra-core/src/auth/usePermissions.ts

```diff
@@ -35,10 +35,11 @@ const emptyParams = {};
  */
 const usePermissions = <Permissions = any, Error = any>(
     params = emptyParams,
-    queryParams: UseQueryOptions<Permissions, Error> = {
-        staleTime: 5 * 60 * 1000,
-    }
+    queryParams: UseQueryOptions<Permissions, Error>
```

Comment:
This was optional before as we provided a default. We must keep it that way
```suggestion
    queryParams?: UseQueryOptions<Permissions, Error>
```